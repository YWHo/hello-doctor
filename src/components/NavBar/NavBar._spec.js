import React from 'react';
import { shallow } from 'enzyme';
import NavBar from './NavBar';

describe('NavBar', () => {
  it('render correctly', () => {
    const wrapper = shallow(<NavBar />);
    expect(wrapper).toMatchSnapshot();
  });
});
