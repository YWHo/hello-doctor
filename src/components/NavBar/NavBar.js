import React from 'react';
import styled from 'styled-components';
import iconGoBack from '../../assets/icon_go_back.svg';
import iconHelp from '../../assets/icon_help.svg';
import vensaLogo from '../../assets/vensa_logo.svg';

const Container = styled.div`
  background-color: #3cb9c0;
  background-image: linear-gradient(to right, #3cb9c0, #2caaca);
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  align-items: center;
  height: 56px;
`;

const ButtonBack = styled.button`
  border: none;
  background-color: Transparent;
  margin-left: 0.4rem;
  margin-top: 0.4rem;
  text-align: center;
`;

const ButtonHelp = styled.button`
  border: none;
  border-right: 5px solid #fff;
  height: 20px;
  background-color: Transparent;
  text-align: center;
`;

export default function NavBar(props) {
  return (
    <Container>
      <ButtonBack>
        <img src={iconGoBack} alt='Go Back' />
      </ButtonBack>
      <img src={vensaLogo} alt='Vensa' />
      <ButtonHelp>
        <img src={iconHelp} alt='Help' />
      </ButtonHelp>
    </Container>
  );
}
