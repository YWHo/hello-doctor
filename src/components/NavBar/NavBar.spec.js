import React from 'react'
import NavBar from './NavBar'

import renderer from 'react-test-renderer'

describe('NavBar', () => {
  it('render correctly', () => {
    const component = renderer.create(<NavBar />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
