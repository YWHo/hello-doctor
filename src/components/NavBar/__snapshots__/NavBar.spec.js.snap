// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar render correctly 1`] = `
<div
  className="sc-bdVaJa iHPEZw"
>
  <button
    className="sc-bwzfXH cruCNk"
  >
    <img
      alt="Go Back"
      src="icon_go_back.svg"
    />
  </button>
  <img
    alt="Vensa"
    src="vensa_logo.svg"
  />
  <button
    className="sc-htpNat jjVWIa"
  >
    <img
      alt="Help"
      src="icon_help.svg"
    />
  </button>
</div>
`;
